src/main/java/fr/insalyon/creatis/gasw/executor/batch/config/json/properties/BatchOptions.java

```diff
@@ -0,0 +1,32 @@
+package fr.insalyon.creatis.gasw.executor.batch.config.json.properties;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter @NoArgsConstructor
+public class BatchOptions {
+    
+    /* value in millis */
+    @JsonProperty(value = "commandExecutionTimeoutInMillis")
+    private int commandExecutionTimeout = 10;
+
+    /* value in millis */
+    @JsonProperty(value = "sshEventTimeoutInMillis")
+    private int sshEventTimeout = 1000;
+
+    @JsonProperty(value = "statusRetry")
+    private int statusRetry = 10;
+
+    /* value in millis */
+    @JsonProperty(value = "statusRetryWaitInMillis")
+    private int statusRetryWait = 10;
+
+    /* By default the system use slurm */
+    @JsonProperty(value = "usePBS")
+    private boolean usePBS = false;
```

Comment:
Wouldn't it be clearer if we had a cluster type property with a list of supported types?